src/index.js

```diff
@@ -1,12 +1,12 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import './index.css';
-import App from './App';
+import Game from './components/Game/Game';
 import reportWebVitals from './reportWebVitals';
 
 ReactDOM.render(
   <React.StrictMode>
-    <App />
+    <Game />
```

Comment:
App is a conventional naming of the root container. So, leave it as it should be.

Reply:
Fixed